### DISCUSSION TITLE
EXEC XINPUT/SETXKBMAP doesn't executed on every start

### DISCUSSION THREAD

**Discussion by dmitrykvasnikov** [i3/i3#5580] [Q&A]
Hello!
I use a few xinput commands in config file (all as exec --no-startup-id): 
  tap to click / natural scroll / layout switch keyboard combination / mapping caps to esc
Every, let's say 5-6 new i3 sessions, those commands just do not run and i have to leave i3 and start again. Although it's not a big deal to check first thing after start if my layout switch working or not, i rather prefer to fix it (if possible, of course)
(AsusVivo laptop, vanilla Arch).

Here is my file (with .txt extention)
[config.txt](https://github.com/i3/i3/files/12038892/config.txt)

**Comment by orestisfl** [comment]
Including your config file would be a good start for someone debugging this

**Reply by dmitrykvasnikov** [reply]
Sorry from newbee, just uploaded it to first post.
Thanks!

**Comment by dipeshsingh253** [comment]
Did you find any solution to this one??
@dmitrykvasnikov
